superset/assets/src/SqlLab/components/ShowSQL.jsx

```diff
@@ -0,0 +1,78 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Badge } from 'react-bootstrap';
+import AceEditor from 'react-ace';
+import 'brace/mode/sql';
+import 'brace/theme/textmate';
+
+import { t } from '@superset-ui/translation';
+
+import Link from './Link';
+import ModalTrigger from '../../components/ModalTrigger';
+
+const propTypes = {
+  tooltipText: PropTypes.string,
+  title: PropTypes.string,
+  sql: PropTypes.string,
+};
+
+const defaultProps = {
+  tooltipText: t('Show SQL'),
+  title: t('SQL statement'),
+  sql: '',
+};
+
+export default class ShowSQL extends React.Component {
+  renderModalBody() {
+    return (
+      <div>
+        <AceEditor
+          mode="sql"
+          theme="textmate"
+          style={{ border: '1px solid #CCC' }}
```

Comment:
NIT, always better to avoid double curlies in props by declaring the object in module scope. Otherwise the object gets recreated every time and leads to more `render` calls if/when using `PureComponent` since it never shallow equals. Also could be CSS.

Reply:
+1